libffi-sys-rs/build/not_msvc.rs

```diff
@@ -61,6 +63,53 @@ pub fn configure_libffi(prefix: PathBuf, build_dir: &Path) {
         }
     }
 
+    let mut c_cfg = cc::Build::new();
+    c_cfg
+        .cargo_metadata(false)
+        .target(&target)
+        .warnings(false)
+        .host(&host);
+    let mut cxx_cfg = cc::Build::new();
+    cxx_cfg
+        .cargo_metadata(false)
+        .cpp(true)
+        .target(&target)
+        .warnings(false)
+        .host(&host);
+    let c_compiler = c_cfg.get_compiler();
+    let cxx_compiler = cxx_cfg.get_compiler();
+
+    let mut cflags = c_compiler.cflags_env();
+    match env::var_os("CFLAGS") {
+        None => (),
+        Some(flags) => {
+            cflags.push(" ");
+            cflags.push(&flags);
+        }
+    }
+    command.env("CFLAGS", cflags);
+
+    let mut cxxflags = cxx_compiler.cflags_env();
+    match env::var_os("CXXFLAGS") {
+        None => (),
+        Some(flags) => {
+            cxxflags.push(" ");
+            cxxflags.push(&flags);
+        }
+    }
+    command.env("CXXFLAGS", cxxflags);
+
+    command.env("CC", c_compiler.path());
+    command.env("CXX", cxx_compiler.path());
+
+    for &(ref k, ref v) in c_compiler.env().iter() {
+        command.env(k, v);
+    }
+
+    for &(ref k, ref v) in cxx_compiler.env().iter() {
+        command.env(k, v);
+    }
+
```

Comment:
Please run `rustfmt` on this file if you haven't already done so, to ensure the formatting is consistent.

Reply:
I've added a rustfmt check job to CI.